edx_solutions_api_integration/users/tests.py

```diff
@@ -47,6 +47,8 @@
 from django.contrib.auth.models import User
 from notification_prefs import NOTIFICATION_PREF_KEY
 
+import before_after
```

Comment:
Import order is not right here 
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#id12